components/form/Conditions.vue

```diff
@@ -27,7 +26,14 @@ export default {
         width: 100,
       },
       STATUS,
-      LAST_HEARTBEAT_TIME,
+      {
```

Comment:
@codyrancher Does this affect the node overview page that also uses this component?

Reply:
@deniseschannon  It looks like the Conditions component is only used by GenericResourceDetail so this wouldn't affect the node overview page. Do you want the node overview page to use this?

Reply:
Yes, we should try to use 1 component across the two pages since it's collecting the same info. 

The spacing in the nodes page is slightly different than the one in this component. @mantis-toboggan-md  can you review between the two to fix the styling to match what is used in your page? I'll leave it between you and @codyrancher to determine who should update the node page :)  https://github.com/rancher/dashboard/blob/master/detail/node.vue